tests/components/application-connector/test/compass-runtime-agent/secret.go

```diff
@@ -0,0 +1,107 @@
+package compass_runtime_agent
+
+import (
+	"context"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	restclient "k8s.io/client-go/rest"
+	"reflect"
+	"testing"
+)
+
+var secretRepos map[string]v1.SecretInterface
+
+type secretData struct {
+	password     string
+	username     string
+	crt          string
+	key          string
+	clientId     string
+	clientSecret string
+}
+
```

Comment:
You are missing mock type for this type 

//go:generate mockery -name=SecretClient